src/test/java/com/aerospike/mapper/BatchLoadTest.java

```diff
@@ -1,26 +1,84 @@
 package com.aerospike.mapper;
 
+import java.util.ArrayList;
+import java.util.List;
+
+import org.junit.Test;
+
 import com.aerospike.mapper.annotations.AerospikeKey;
 import com.aerospike.mapper.annotations.AerospikeRecord;
 import com.aerospike.mapper.annotations.ParamFrom;
+import com.aerospike.mapper.tools.AeroMapper;
 
 public class BatchLoadTest extends AeroMapperBaseTest {
-	@AerospikeRecord(namespace = "test", set = "C")
-	public static class C {
+	@AerospikeRecord(namespace = "test", set = "B")
+	public static class B {
 		@AerospikeKey
 		public int id;
 		public String name;
 		
-		public C(@ParamFrom("id") int id, @ParamFrom("name") String name) {
+		public B(@ParamFrom("id") int id, @ParamFrom("name") String name) {
 			super();
 			this.id = id;
 			this.name = name;
 		}
 	}
 	
-	@AerospikeRecord(namespace = "test", set = "B")
-	public static class B {
+	@AerospikeRecord(namespace = "test", set = "A")
+	public static class A {
 		@AerospikeKey
 		public int id;
+		public String name;
+		public List<B> data;
+		public A(int id, String name) {
+			super();
+			this.id = id;
+			this.name = name;
+			data = new ArrayList<>();
+		}
+		
+		public A addB(B b) {
+			data.add(b);
+			return this;
+		}
+		
+		public A() {
+			super();
+		}
 	}
+	
+	@Test
+	public void testBatchLoad() {
+		AeroMapper mapper = new AeroMapper.Builder(client).build();
+		
+		B[] bees = new B[100];
```

Comment:
Im not sure why we need this complex calculation if all we want to create is some A's that contains some B's in order to check that the batch read works.

This is an example of a some of what we get after running this:

A id:107
A name:A-7
A data size:56
A Bees:B-17 A Bees:B-18 A Bees:B-19 A Bees:B-20 A Bees:B-21 A Bees:B-22 A Bees:B-23 A Bees:B-24 A Bees:B-25 A Bees:B-26 A Bees:B-27 A Bees:B-28 A Bees:B-29 A Bees:B-30 A Bees:B-31 A Bees:B-32 A Bees:B-33 A Bees:B-34 A Bees:B-35 A Bees:B-36 A Bees:B-37 A Bees:B-38 A Bees:B-39 A Bees:B-40 A Bees:B-41 A Bees:B-42 A Bees:B-43 A Bees:B-44 A Bees:B-45 A Bees:B-46 A Bees:B-47 A Bees:B-48 A Bees:B-49 A Bees:B-50 A Bees:B-51 A Bees:B-52 A Bees:B-53 A Bees:B-54 A Bees:B-55 A Bees:B-56 A Bees:B-57 A Bees:B-58 A Bees:B-59 A Bees:B-60 A Bees:B-61 A Bees:B-62 A Bees:B-63 A Bees:B-64 A Bees:B-65 A Bees:B-66 A Bees:B-67 A Bees:B-68 A Bees:B-69 A Bees:B-70 A Bees:B-71 A Bees:B-72  

A id:108
A name:A-8
A data size:64
A Bees:B-18 A Bees:B-19 A Bees:B-20 A Bees:B-21 A Bees:B-22 A Bees:B-23 A Bees:B-24 A Bees:B-25 A Bees:B-26 A Bees:B-27 A Bees:B-28 A Bees:B-29 A Bees:B-30 A Bees:B-31 A Bees:B-32 A Bees:B-33 A Bees:B-34 A Bees:B-35 A Bees:B-36 A Bees:B-37 A Bees:B-38 A Bees:B-39 A Bees:B-40 A Bees:B-41 A Bees:B-42 A Bees:B-43 A Bees:B-44 A Bees:B-45 A Bees:B-46 A Bees:B-47 A Bees:B-48 A Bees:B-49 A Bees:B-50 A Bees:B-51 A Bees:B-52 A Bees:B-53 A Bees:B-54 A Bees:B-55 A Bees:B-56 A Bees:B-57 A Bees:B-58 A Bees:B-59 A Bees:B-60 A Bees:B-61 A Bees:B-62 A Bees:B-63 A Bees:B-64 A Bees:B-65 A Bees:B-66 A Bees:B-67 A Bees:B-68 A Bees:B-69 A Bees:B-70 A Bees:B-71 A Bees:B-72 A Bees:B-73 A Bees:B-74 A Bees:B-75 A Bees:B-76 A Bees:B-77 A Bees:B-78 A Bees:B-79 A Bees:B-80 A Bees:B-81  

A id:109
A name:A-9
A data size:72
A Bees:B-19 A Bees:B-20 A Bees:B-21 A Bees:B-22 A Bees:B-23 A Bees:B-24 A Bees:B-25 A Bees:B-26 A Bees:B-27 A Bees:B-28 A Bees:B-29 A Bees:B-30 A Bees:B-31 A Bees:B-32 A Bees:B-33 A Bees:B-34 A Bees:B-35 A Bees:B-36 A Bees:B-37 A Bees:B-38 A Bees:B-39 A Bees:B-40 A Bees:B-41 A Bees:B-42 A Bees:B-43 A Bees:B-44 A Bees:B-45 A Bees:B-46 A Bees:B-47 A Bees:B-48 A Bees:B-49 A Bees:B-50 A Bees:B-51 A Bees:B-52 A Bees:B-53 A Bees:B-54 A Bees:B-55 A Bees:B-56 A Bees:B-57 A Bees:B-58 A Bees:B-59 A Bees:B-60 A Bees:B-61 A Bees:B-62 A Bees:B-63 A Bees:B-64 A Bees:B-65 A Bees:B-66 A Bees:B-67 A Bees:B-68 A Bees:B-69 A Bees:B-70 A Bees:B-71 A Bees:B-72 A Bees:B-73 A Bees:B-74 A Bees:B-75 A Bees:B-76 A Bees:B-77 A Bees:B-78 A Bees:B-79 A Bees:B-80 A Bees:B-81 A Bees:B-82 A Bees:B-83 A Bees:B-84 A Bees:B-85 A Bees:B-86 A Bees:B-87 A Bees:B-88 A Bees:B-89 A Bees:B-90

Im not sure why we require this specific calculation (*8?, i+10+j?) if there is no reason to use these exact numbers or pattern (each increasing by 8 starting the last element of the previous one) then I think we should make it a bit more simple and readable if someone else will need to modify this test in the future :)